pkg/tasks/local/ticket.go

```diff
@@ -99,6 +98,7 @@ func (m *localCreateTicket) Process(ctx context.Context, task *v1.Task, cc types
 	ticketRequestBody.CustomFields = cfs
 	ticketReq := &v2.TicketsServiceCreateTicketRequest{
 		Request: ticketRequestBody,
+		Schema:  schema.GetSchema(),
```

Comment:
we have to add this to `HandleTask` in `pkg/tasks/c1api/create_ticket.go` also right?

Reply:
Good call. Updated in 3696614